tests/tools/singlestore_search_tool_test.py

```diff
@@ -0,0 +1,346 @@
+import os
+from typing import Generator
+
+import pytest
+
+try:
+    from singlestoredb import connect
+    from singlestoredb.server import docker
+
+    SINGLESTORE_AVAILABLE = True
+except ImportError:
+    SINGLESTORE_AVAILABLE = False
+
+from crewai_tools import SingleStoreSearchTool, SingleStoreSearchToolSchema
+
+# Skip all tests if SingleStore dependencies are not available
+pytestmark = pytest.mark.skipif(
+    not SINGLESTORE_AVAILABLE, reason="SingleStore dependencies not available"
+)
```

Comment:
Please remove this skipped pytest test. During development or when running tests, we should install deps using  `--all-extras` flag.